src/igvfd/searches/configs/prediction_set.py

```diff
@@ -7,6 +7,48 @@
 def prediction_set():
     return {
         'facets': {
+            'file_set_type': {
+                'title': 'File Set Type',
+            },
+            'scope': {
+                'title': 'Scope',
+            },
+            'assessed_genes': {
+                'title': 'Assessed Genes',
+            },
+            'donors.taxa': {
+                'title': 'Taxa',
+            },
+            'samples.classifications': {
+                'title': 'Classification',
+            },
+            'samples.sample_terms.term_name': {
+                'title': 'Sample',
+            },
+            'samples.targeted_sample_terms.term_name': {
```

Comment:
I think this should be `samples.targeted_sample_term.term_name`. It’s spelled this way in the facet groups.

Reply:
done